go/pkg/bertyprotocol/api_event.go

```diff
@@ -56,15 +57,15 @@ func (s *service) GroupMetadataList(req *bertytypes.GroupMetadataList_Request, s
 		newEvents = cg.MetadataStore().Subscribe(sub.Context())
 	}
 
-	// Subscribe to previous metadata events and stream them if requested
-	if !req.SinceNow {
+	listPreviousMessages := func() error {
+		var previousEvents <-chan *bertytypes.GroupMetadataEvent
 		previousEvents, err = cg.MetadataStore().ListEvents(sub.Context(), req.SinceID, req.UntilID, req.ReverseOrder)
```

Comment:
```suggestion
		previousEvents, err := cg.MetadataStore().ListEvents(sub.Context(), req.SinceID, req.UntilID, req.ReverseOrder)
```
Looks cleaner I think :)